frontend/src/components/Course/Course.tsx

```diff
@@ -9,36 +9,38 @@ import {
   CourseTitle,
   CourseDescription,
   ButtonsContainer,
-  Divider,
 } from './styled';
 
 interface Props {
   title: string;
   description: string;
-  language: string;
   duration: string;
   lessons: string;
-  link: string;
+  styleProps?: {
+    fontSize?: number;
+    lineHeight?: number;
+    size?: string;
+  };
 }
 
 const CourseItem: React.FC<Props> = ({
   title,
   description,
-  language,
   duration,
   lessons,
-  link,
+  styleProps,
   children,
 }) => (
-  <CourseContainer>
+  <CourseContainer size={styleProps?.size}>
     <ImageWrapper>
       <Image width={300} height={200} />
```

Comment:
We Will fix this magic number in future.

Reply:
removed